### PR TITLE
Refactor to remove InputParams struct

### PR DESCRIPTION
**Pull request by howardjohn**
This struct is no longer used beyond tests, after we stopped using
'Plugins' concept. This fully cleans up the struct from tests that were
still using it.

**Please provide a description of this PR:**

### CODE REVIEW

File: pilot/pkg/networking/plugin/plugin.go [hunk 1]
```diff
@@ -13,44 +13,3 @@
 // limitations under the License.
 
 package plugin
```
Comment (hzxuzhonghu) [inline pilot/pkg/networking/plugin/plugin.go:15]: why keep a blank plugin

Comment (howardjohn) [reply pilot/pkg/networking/plugin/plugin.go:15]: good point, will remove it

Comment (ericvn) [reply pilot/pkg/networking/plugin/plugin.go:15]: LGTM after this change.

**Review by ramaraochavali — APPROVED** [review]

**Review by ericvn — APPROVED** [review]

**Review by lei-tang — APPROVED** [review]

**Pull request merged**